test/library/packages/UnrolledLinkedList/types/PREDIFF

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Ignores line numbers printed upon compilerError in Sort.chpl
+# Note: This is a work around for a bug. The line numbers should not be printed
```

Comment:
There are no lines from Sort.chpl in any of these .good files (and there are no .bad files).  Was this referencing something in Sort.chpl that's not happening any more, or was this copied from somewhere that did have errors from Sort.chpl to worry about?